docs/framework/docker/console.md

```diff
@@ -0,0 +1,139 @@
+
+Moving your console application is a matter of a few steps.
+
+1. Build the application
+1. Creating a Dockerfile for the image
+1. Process to build and run the Docker container
+
+## Building the application
+Typically console applications are distributed through an installer, FTP, or File Share deployment. Here the assets need to be compiled and staged to a location that can be used when the Docker image is created.
+
+In **build.ps1** the script uses MSBuild to compile the application with a configuration of *Release* and publishes the assets to the *publish* folder.
+
+```
+function Invoke-MSBuild ([string]$MSBuildPath, [string]$MSBuildParameters) {
+    Invoke-Expression "$MSBuildPath $MSBuildParameters"
+}
+
+Invoke-MSBuild -MSBuildPath "MSBuild.exe" -MSBuildParameters ".\ConsoleRandomAnswerGenerator.csproj /p:OutputPath=.\publish /p:Configuration=Release"
+```
+
+## Creating the Dockerfile
+The base image used for a Console .NET Framework applications is `microsoft/windowsservercore`, publically available on [Docker Hub](https://hub.docker.com/r/microsoft/windowsservercore/). The image is the minimal installation of Windows Server 2016 and serves as the base OS image for Windows containers.
+
+```
+FROM microsoft/windowsservercore
+ADD publish/ /
+ENTRYPOINT ConsoleRandomAnswerGenerator.exe
+```
+
+The `ADD` in the file copies the application assets from the **publish** folder to root folder of the container and last; setting the `ENTRYPOINT` of the image states that this is the command or application that will run when the container starts. 
+
+## Creating the image
+Adding the following code to the **build.ps1** script, when run the Docker image called `console-random-answer-generator` is created after the compilation is complete.
+
+```
+$ImageName="console-random-answer-generator"
+
+function Invoke-Docker-Build ([string]$ImageName, [string]$ImagePath, [string]$DockerBuildArgs = "") {
+    echo "docker build -t $ImageName $ImagePath $DockerBuildArgs"
+    Invoke-Expression "docker build -t $ImageName $ImagePath $DockerBuildArgs"
+}
+
+Invoke-Docker-Build -ImageName $ImageName -ImagePath "."
+```
+
+When the build is complete, using the `docker images` command from a command line or PowerShell prompt; you'll see that the image is created and ready to be run.
+
+```
+REPOSITORY                        TAG                 IMAGE ID            CREATED             SIZE
+console-random-answer-generator   latest              8f7c807db1b5        8 seconds ago       7.33 GB
+```
+
+## Running the container
+You can start the container from the command line using the Docker commands.
+
+```
+docker run console-random-answer-generator "Are you a square container?"
```

Comment:
I could see people wnodering how they would supply multiple parameters to their console application. Perhaps adding a "Note" section with an example would be helpful?